README.md

```diff
@@ -554,6 +554,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 - [Textpattern](http://textpattern.com/) - Flexible, elegant and easy-to-use CMS. ([Demo](http://textpattern.co/demo), [Source Code](https://github.com/textpattern/textpattern)) `GPL-2.0` `PHP`
 - [TYPO3](https://typo3.org/) - Powerful and advanced CMS with a large community. ([Source Code](https://github.com/TYPO3/TYPO3.CMS)) `GPL-2.0` `PHP`
 - [Umbraco](https://umbraco.com/) - The friendly CMS. Free and open source with an amazing community. ([Source Code](https://github.com/umbraco/Umbraco-CMS)) `MIT` `.NET`
+- [Void](https://github.com/josephernest/void) - Flat file CMS with pages + blog articles, in less than 100 lines of PHP code. ([Demo](https://gget.it/void/), [Source Code](https://github.com/josephernest/void)) `MIT` `PHP`
```

Comment:
Remove `, [Source Code](https://github.com/josephernest/void)` its not needed since it the same as the main link.

Reply:
@n8225 I don't know how to modify code in an existing PR, could you remove it and merge it? Thanks in advance!